### DISCUSSION TITLE
ModuleNotFoundErro: No module named 'tf.compat'; 'tf' is not a package

### DISCUSSION THREAD

**Discussion by The132** [bulletphysics/bullet3#3770] [General]
after the code update "Replace tensorflow with tf.compat.v1"，I meet a problem when run "python3 minitaur_trotting_env_example.py",The terminal presents the following error "ModuleNotFoundErro: No module named 'tf.compat'; 'tf' is not a package". how can I solve it.

**Comment by erwincoumans** [comment]
It was a commit that doesn't support backwards compatibility. We need to add try/except statements instead. TensorFlow keeps on breaking the API.
~~~~~~~~~~~~~~~~~~~
try:
  import tensorflow.compat.v1 as tf
except Exception:
  import tensorflow as tf
~~~~~~~~~~~~~~~~~~~

**Comment by The132** [comment]
Thanks for your answer and hope this work can be completed as soon as possible，thank you.

**Comment by erwincoumans** [comment]
Sounds good, we'll await your contribution.
